mesonbuild/dependencies/dev.py

```diff
@@ -56,7 +56,9 @@ class GTestDependencySystem(SystemDependency):
     def __init__(self, name: str, environment: 'Environment', kwargs: T.Dict[str, T.Any]) -> None:
         super().__init__(name, environment, kwargs, language='cpp')
         self.main = kwargs.get('main', False)
-        self.src_dirs = ['/usr/src/gtest/src', '/usr/src/googletest/googletest/src']
+
+        sysroot = environment.properties[self.for_machine].get_sys_root() or ""
+        self.src_dirs = [sysroot + '/usr/src/gtest/src', sysroot + '/usr/src/googletest/googletest/src']
```

Comment:
Inconsistent quoting style, consolidate on single quotes as used elsewhere.

Reply:
Apologies, my double-quote muscle memory is strong. I find writing meson.build files quite painful :)